themes/default/layouts/page/pricing.html

```diff
@@ -61,9 +61,9 @@ <h2 class="text-3xl text-center mt-10 mb-0 lg:hidden">Pulumi Service</h2>
                             </li>
                         </ul>
                     </div>
-                    <div class="mt-6 lg:mt-16 lg:pt-3">
-                        <a href="{{ relref . "/docs/get-started" }}" class="btn btn-purple px-6 py-3 inline-block lg:hidden">GET STARTED</a>
-                        <a href="{{ relref . "/docs/get-started" }}" class="btn btn-purple-transparent px-6 py-3 hidden lg:inline-block">GET STARTED</a>
+                    <div class="w-full text-center hidden lg:inline-block lg:mt-10 lg:pt-1">
```

Comment:
```suggestion
                    <div class="w-full text-center hidden lg:inline-block lg:mt-10 lg:pt-2">
```

That should make the buttons align. CSS be CSS sometimes